src/modules/control_allocator/ControlAllocator.cpp

```diff
@@ -316,6 +320,17 @@ ControlAllocator::Run()
 		_actuator_effectiveness->setFlightPhase(flight_phase);
 	}
 
+	if (_airspeed_sub.updated()) {
+		airspeed_validated_s airspeed = {};
+
+		_airspeed_sub.update(&airspeed);
+
+		if (PX4_ISFINITE(airspeed.indicated_airspeed_m_s)) {
+			const float airspeed_scaling = _param_airspeed_ias_trim.get() / math::max(airspeed.indicated_airspeed_m_s, 3.0f);
+			_actuator_effectiveness->updateAirspeedScaling(airspeed_scaling);
```

Comment:
We may want to move that up to the effectiveness interface. Something like setAirspeed(float airspeed).
Current actuator output might also be useful for tiltrotors.

Reply:
@jlecoeur Agreed, I'll change that.